### DISCUSSION TITLE
store.steampowered.com

### DISCUSSION THREAD

**Discussion by Fan4eGGG** [catppuccin/catppuccin#2602] [Port Requests]
### Will you make the port?

- [ ] Yes, I'm willing to make the port

### Any additional comments?

I like ur themes and would love to see Steam (_steamcommunity.com, store.steampowered.com_) support!
tnx

### Submission Guidelines

- [X] I have read and followed Catppuccin's submission guidelines.

**Comment by uncenter** [comment]
See https://github.com/catppuccin/steam.

**Reply by Fan4eGGG** [reply]
> See https://github.com/catppuccin/steam.

i mean theme for site, not for app
